lib/tasks/reminder.rake

```diff
@@ -0,0 +1,21 @@
+namespace :reminder do
```

Comment:
might make sense to have a more generic namespace, maybe something like `email` ?